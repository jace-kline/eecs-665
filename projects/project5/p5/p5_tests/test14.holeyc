void main() {
    bool b;
    b++;
    b--;
}