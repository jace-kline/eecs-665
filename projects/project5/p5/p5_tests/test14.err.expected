FATAL [7,6]: Arithmetic operator applied to incompatible operands
FATAL [8,6]: Arithmetic operator applied to incompatible operands
Type Analysis Failed
